docs/api/event-brokers.rst

```diff
@@ -8,9 +8,23 @@ Event Brokers
 
 .. edit-link::
 
-Rasa Core allows you to stream events to a message broker. The event broker
-emits events into the event queue. It becomes part of the ``TrackerStore``
-which you use when starting an ``Agent`` or launch ``rasa.core.run``.
+When running a Rasa Open Source assistant in production, you might want to
```

Comment:
I'd suggest making this intro a bit clearer by making 'event broker' the subject of the first sentence. It feels a bit buried by the context. Something like this?

An event broker allows you to connect your assistant to other services that need access to data coming in from conversations. For example, you could connect your assistant to Rasa X to review and annotate conversations or forward messages to an external analytics service.

Reply:
Love it